src/components/calendar/calendar.js

```diff
@@ -325,8 +324,16 @@
     switch (event.which) {
       case keyCode.RIGHT_ARROW: return dateUtil.incrementDays(this.displayDate, 1);
       case keyCode.LEFT_ARROW: return dateUtil.incrementDays(this.displayDate, -1);
-      case keyCode.DOWN_ARROW: return dateUtil.incrementDays(this.displayDate, 7);
-      case keyCode.UP_ARROW: return dateUtil.incrementDays(this.displayDate, -7);
+      case keyCode.DOWN_ARROW:
+        if (event.metaKey) {
```

Comment:
I think this would be cleaner with a ternary:
```js
return event.metaKey ?
    dateUtil.incrementMonths(this.displayDate, 1) :
   dateUtil.incrementDays(this.displayDate, 7);
```